### PR TITLE
Zombies for Houses

### PR DESCRIPTION
**Pull request by Maleclypse**
#### Summary
Balance "Zombies for empty houses"

#### Purpose of change
I don't mind houses lacking zombies but I checked and a lot of more recent house additions had no chance to ever spawn zombies.  This travesty cannot stand.   

#### Describe the solution
Adds zombies to houses lacking them.

#### Describe alternatives you've considered
None

### CODE REVIEW

**Comment by AudBobb** [comment]
damn, someone upset the github bot :p

**Comment by Maleclypse** [comment]
> damn, someone upset the github bot :p

Hehe yep that is what happens when you copy the same broken line 132 times

**Comment by Procyonae** [comment]
At least most of these have spawns defined in their OMT definition or by inheritance like
```
  {
    "type": "overmap_terrain",
    "abstract": "generic_city_house",
    "copy-from": "generic_city_building",
    "name": "house",
    "color": "light_green",
    "see_cost": "high",
    "spawns": { "group": "GROUP_VANILLA", "population": [ 1, 4 ], "chance": 80 },
    "extend": { "flags": [ "GENERIC_LOOT" ] }
  },
```

File: data/json/mapgen/house/house_detatched5.json [hunk 1]
```diff
@@ -149,6 +150,7 @@
         "                        "
       ],
       "palettes": [ "basement_empty" ],
+      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.05  } ],
```
Comment (Maleclypse) [inline data/json/mapgen/house/house_detatched5.json:153]:
```suggestion
      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.05 } ],
```

File: data/json/mapgen/house/house_detatched6.json [hunk 2]
```diff
@@ -182,6 +182,7 @@
           ]
         }
       },
+      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.05  } ],
```
Comment (Maleclypse) [inline data/json/mapgen/house/house_detatched6.json:185]:
```suggestion
      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.05 } ],
```

File: data/json/mapgen/house/house_detatched7.json [hunk 3]
```diff
@@ -55,6 +55,7 @@
         "&": "t_sandbox"
       },
       "place_nested": [ { "chunks": [ "rolling_trash_can_1x1" ], "x": 7, "y": 0 } ],
+      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.2  } ],
```
Comment (Maleclypse) [inline data/json/mapgen/house/house_detatched7.json:58]:
```suggestion
      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.2 } ],
```

File: data/json/mapgen/house/house_detatched7.json [hunk 4]
```diff
@@ -142,6 +143,7 @@
         "                        "
       ],
       "palettes": [ "basement_empty" ],
+      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.05  } ],
```
Comment (Maleclypse) [inline data/json/mapgen/house/house_detatched7.json:146]:
```suggestion
      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.05 } ],
```

File: data/json/mapgen/house/house_duplex.json [hunk 5]
```diff
@@ -50,6 +50,7 @@
         { "chunks": [ "rolling_trash_can_1x1" ], "x": 18, "y": 1 }
       ],
       "furniture": { "!": "f_region_flower" },
+      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.1  } ],
```
Comment (Maleclypse) [inline data/json/mapgen/house/house_duplex.json:53]:
```suggestion
      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.1 } ],
```

File: data/json/mapgen/house/house_duplex10.json [hunk 6]
```diff
@@ -49,6 +49,7 @@
         "?": "t_screen_door_c"
       },
       "furniture": { "!": "f_region_flower" },
+      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.1  } ],
```
Comment (Maleclypse) [inline data/json/mapgen/house/house_duplex10.json:52]:
```suggestion
      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.1 } ],
```

File: data/json/mapgen/house/house_duplex11.json [hunk 7]
```diff
@@ -52,6 +52,7 @@
         { "chunks": [ "rolling_trash_can_1x1" ], "x": 4, "y": 1 },
         { "chunks": [ "rolling_trash_can_1x1" ], "x": 20, "y": 1 }
       ],
+      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.1  } ],
```
Comment (Maleclypse) [inline data/json/mapgen/house/house_duplex11.json:55]:
```suggestion
      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.1 } ],
```

File: data/json/mapgen/house/house_duplex2.json [hunk 8]
```diff
@@ -49,6 +49,7 @@
         { "chunks": [ "rolling_trash_can_1x1" ], "x": 6, "y": 0 },
         { "chunks": [ "rolling_trash_can_1x1" ], "x": 16, "y": 0 }
       ],
+      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.1  } ],
```
Comment (Maleclypse) [inline data/json/mapgen/house/house_duplex2.json:52]:
```suggestion
      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.1 } ],
```

File: data/json/mapgen/house/house_duplex3.json [hunk 9]
```diff
@@ -49,6 +49,7 @@
         { "chunks": [ "rolling_trash_can_1x1" ], "x": 5, "y": 1 },
         { "chunks": [ "rolling_trash_can_1x1" ], "x": 13, "y": 1 }
       ],
+      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.1  } ],
```
Comment (Maleclypse) [inline data/json/mapgen/house/house_duplex3.json:52]:
```suggestion
      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.1 } ],
```

File: data/json/mapgen/house/house_duplex4.json [hunk 10]
```diff
@@ -49,6 +49,7 @@
         { "chunks": [ "rolling_trash_can_1x1" ], "x": 6, "y": 0 },
         { "chunks": [ "rolling_trash_can_1x1" ], "x": 12, "y": 0 }
       ],
+      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.1  } ],
```
Comment (Maleclypse) [inline data/json/mapgen/house/house_duplex4.json:52]:
```suggestion
      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.1 } ],
```

File: data/json/mapgen/house/house_duplex5.json [hunk 11]
```diff
@@ -51,6 +51,7 @@
         { "chunks": [ "rolling_trash_can_1x1" ], "x": 3, "y": 1 },
         { "chunks": [ "rolling_trash_can_1x1" ], "x": 12, "y": 1 }
       ],
+      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.1  } ],
```
Comment (Maleclypse) [inline data/json/mapgen/house/house_duplex5.json:54]:
```suggestion
      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.1 } ],
```

File: data/json/mapgen/house/house_duplex6.json [hunk 12]
```diff
@@ -62,6 +62,7 @@
         { "chunks": [ "rolling_trash_can_1x1" ], "x": 3, "y": 1 },
         { "chunks": [ "rolling_trash_can_1x1" ], "x": 12, "y": 1 }
       ],
+      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.1  } ],
```
Comment (Maleclypse) [inline data/json/mapgen/house/house_duplex6.json:65]:
```suggestion
      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.1 } ],
```

File: data/json/mapgen/house/house_duplex7.json [hunk 13]
```diff
@@ -62,6 +62,7 @@
         { "chunks": [ "rolling_trash_can_1x1" ], "x": 4, "y": 1 },
         { "chunks": [ "rolling_trash_can_1x1" ], "x": 12, "y": 1 }
       ],
+      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.1  } ],
```
Comment (Maleclypse) [inline data/json/mapgen/house/house_duplex7.json:65]:
```suggestion
      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.1 } ],
```

File: data/json/mapgen/house/house_duplex9.json [hunk 14]
```diff
@@ -54,6 +54,7 @@
         { "group": "mischw", "x": [ 13, 20 ], "y": [ 2, 11 ], "chance": 80, "repeat": [ 1, 4 ] },
         { "item": "television", "x": 17, "y": 13, "chance": 100 }
       ],
+      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.1  } ],
```
Comment (Maleclypse) [inline data/json/mapgen/house/house_duplex9.json:57]:
```suggestion
      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.1 } ],
```

File: data/json/mapgen/house/house_fence01.json [hunk 15]
```diff
@@ -34,6 +34,7 @@
       ],
       "palettes": [ "standard_domestic_palette", "standard_domestic_basement_palette" ],
       "terrain": { "u": "t_carpet_purple", "X": "t_carpet_purple", "c": "t_carpet_purple", "_": "t_carpet_purple" },
+      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.1  } ],
```
Comment (Maleclypse) [inline data/json/mapgen/house/house_fence01.json:37]:
```suggestion
      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.1 } ],
```

File: data/json/mapgen/house/house_fence01.json [hunk 16]
```diff
@@ -71,7 +72,8 @@
         "^^^^^^^^^^^^^^^^^^^^^^^^"
       ],
       "palettes": [ "standard_domestic_palette", "standard_domestic_basement_palette" ],
-      "furniture": { "!": "f_desk", "$": "f_wardrobe", "%": "f_dresser", "&": "f_mannequin", "~": "f_sink" }
+      "furniture": { "!": "f_desk", "$": "f_wardrobe", "%": "f_dresser", "&": "f_mannequin", "~": "f_sink" },
+      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.1  } ]
```
Comment (Maleclypse) [inline data/json/mapgen/house/house_fence01.json:76]:
```suggestion
      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.1 } ]
```

File: data/json/mapgen/house/house_fence01.json [hunk 17]
```diff
@@ -117,6 +119,7 @@
         "!": "t_carpet_purple",
         "A": "t_carpet_purple"
       },
+      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.1  } ],
```
Comment (Maleclypse) [inline data/json/mapgen/house/house_fence01.json:122]:
```suggestion
      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.1 } ],
```

File: data/json/mapgen/house/house_fence01.json [hunk 18]
```diff
@@ -190,6 +193,7 @@
       "furniture": { "k": "f_chair", "0": "f_region_flower" },
       "items": { "~": { "item": "trash", "chance": 3 }, ")": { "item": "clutter_bathroom", "chance": 5 } },
       "place_loot": [ { "item": "television", "x": 10, "y": 11, "chance": 100 }, { "item": "lawnmower", "x": 2, "y": 3, "chance": 15 } ],
+      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.1  } ],
```
Comment (Maleclypse) [inline data/json/mapgen/house/house_fence01.json:196]:
```suggestion
      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.1 } ],
```

File: data/json/mapgen/house/house_fence02.json [hunk 19]
```diff
@@ -106,6 +107,7 @@
         "q": "t_floor_noroof",
         "A": { "param": "linoleum_color_kitchen", "fallback": "t_linoleum_gray" }
       },
+      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.1  } ],
```
Comment (Maleclypse) [inline data/json/mapgen/house/house_fence02.json:110]:
```suggestion
      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.1 } ],
```

File: data/json/mapgen/house/house_fence03.json [hunk 20]
```diff
@@ -48,6 +48,7 @@
         "g": "t_thconc_floor",
         "_": { "param": "linoleum_color_bathroom", "fallback": "t_linoleum_gray" }
       },
+      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.1  } ],
```
Comment (Maleclypse) [inline data/json/mapgen/house/house_fence03.json:51]:
```suggestion
      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.1 } ],
```

File: data/json/mapgen/house/house_fence03.json [hunk 21]
```diff
@@ -88,6 +89,7 @@
       ],
       "terrain": { "{": "t_chainfence", ")": "t_chaingate_c", "=": "t_carpet_purple", "`": "t_carpet_yellow" },
       "furniture": { "(": "f_rack", "$": "f_table" },
+      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.1  } ],
```
Comment (Maleclypse) [inline data/json/mapgen/house/house_fence03.json:92]:
```suggestion
      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.1 } ],
```

File: data/json/mapgen/house/house_fence04.json [hunk 22]
```diff
@@ -52,6 +52,7 @@
         "&": "t_floor_noroof"
       },
       "furniture": { "&": "f_table", "?": [ "f_indoor_plant", "f_indoor_plant_y" ] },
+      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.1  } ],
```
Comment (Maleclypse) [inline data/json/mapgen/house/house_fence04.json:55]:
```suggestion
      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.1 } ],
```

File: data/json/mapgen/house/house_fence04.json [hunk 23]
```diff
@@ -181,6 +182,7 @@
         "0": { "item": "hydro", "chance": 50 },
         "&": [ { "item": "farming_seeds", "chance": 35 }, { "item": "farming_tools", "chance": 50 } ]
       },
+      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.1  } ],
```
Comment (Maleclypse) [inline data/json/mapgen/house/house_fence04.json:185]:
```suggestion
      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.1 } ],
```

File: data/json/mapgen/house/house_fence05.json [hunk 24]
```diff
@@ -33,6 +33,7 @@
         "^^^^^^^^^^^^^^^^^^^^^^^^",
         "^^^^^^^^^^^^^^^^^^^^^^^^"
       ],
+      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.1  } ],
```
Comment (Maleclypse) [inline data/json/mapgen/house/house_fence05.json:36]:
```suggestion
      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.1 } ],
```

File: data/json/mapgen/house/house_fence05.json [hunk 25]
```diff
@@ -89,6 +90,7 @@
         "Q": "t_null",
         "`": "t_screen_door_c"
       },
+      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.1  } ],
```
Comment (Maleclypse) [inline data/json/mapgen/house/house_fence05.json:93]:
```suggestion
      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.1 } ],
```

File: data/json/mapgen/house/house_fence05.json [hunk 26]
```diff
@@ -135,6 +137,7 @@
         "$": { "item": "table_foyer", "chance": 50 },
         "!": [ { "item": "dress_shop", "chance": 50, "repeat": [ 1, 2 ] }, { "item": "jackets", "chance": 30, "repeat": [ 1, 2 ] } ]
       },
+      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.1  } ],
```
Comment (Maleclypse) [inline data/json/mapgen/house/house_fence05.json:140]:
```suggestion
      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.1 } ],
```

File: data/json/mapgen/house/house_garage.json [hunk 27]
```diff
@@ -159,6 +159,7 @@
         "                        ",
         "                        "
       ],
+      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.05  } ],
```
Comment (Maleclypse) [inline data/json/mapgen/house/house_garage.json:162]:
```suggestion
      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.05 } ],
```

File: data/json/mapgen/house/house_garage2.json [hunk 28]
```diff
@@ -132,6 +132,7 @@
         "                        "
       ],
       "palettes": [ "basement_empty" ],
+      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.02  } ],
```
Comment (Maleclypse) [inline data/json/mapgen/house/house_garage2.json:135]:
```suggestion
      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.02 } ],
```

File: data/json/mapgen/house/house_garage3.json [hunk 29]
```diff
@@ -124,6 +124,7 @@
         "                        "
       ],
       "palettes": [ "basement_empty" ],
+      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.01  } ],
```
Comment (Maleclypse) [inline data/json/mapgen/house/house_garage3.json:127]:
```suggestion
      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.01 } ],
```

File: data/json/mapgen/house/house_garage4.json [hunk 30]
```diff
@@ -120,6 +120,7 @@
         "                        "
       ],
       "palettes": [ "basement_empty" ],
+      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.01  } ],
```
Comment (Maleclypse) [inline data/json/mapgen/house/house_garage4.json:123]:
```suggestion
      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.01 } ],
```

File: data/json/mapgen/house/house_garage5.json [hunk 31]
```diff
@@ -116,6 +116,7 @@
         "                        "
       ],
       "palettes": [ "basement_empty" ],
+      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.01  } ],
```
Comment (Maleclypse) [inline data/json/mapgen/house/house_garage5.json:119]:
```suggestion
      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.01 } ],
```

File: data/json/mapgen/house/house_garage6.json [hunk 32]
```diff
@@ -152,6 +152,7 @@
         "                        "
       ],
       "palettes": [ "basement_empty" ],
+      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.01  } ],
```
Comment (Maleclypse) [inline data/json/mapgen/house/house_garage6.json:155]:
```suggestion
      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.01 } ],
```

File: data/json/mapgen/s_bookstore.json [hunk 33]
```diff
@@ -76,6 +76,7 @@
         ],
         "%": { "item": "cash_register_random", "chance": 100 }
       },
+      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.2  } ],
```
Comment (Maleclypse) [inline data/json/mapgen/s_bookstore.json:79]:
```suggestion
      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.2 } ],
```

File: data/json/mapgen/s_bookstore.json [hunk 34]
```diff
@@ -265,6 +266,7 @@
         "S": "f_sink"
       },
       "toilets": { "t": {  } },
+      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.2  } ],
```
Comment (Maleclypse) [inline data/json/mapgen/s_bookstore.json:269]:
```suggestion
      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.2 } ],
```

File: data/json/mapgen/house/house_detatched1.json [hunk 35]
```diff
@@ -33,6 +33,7 @@
         ".......%...%........%..."
       ],
       "palettes": [ "domestic_general_and_variant_palette" ],
+      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "density": 0.2 } ],
```
Comment (Maleclypse) [inline data/json/mapgen/house/house_detatched1.json:36]:
```suggestion
```

**Comment by Maleclypse** [comment]
> At least most of these already have spawns defined in their OMT definition or by inheritance like
> 
> ```
>   {
>     "type": "overmap_terrain",
>     "abstract": "generic_city_house",
>     "copy-from": "generic_city_building",
>     "name": "house",
>     "color": "light_green",
>     "see_cost": "high",
>     "spawns": { "group": "GROUP_VANILLA", "population": [ 1, 4 ], "chance": 80 },
>     "extend": { "flags": [ "GENERIC_LOOT" ] }
>   },
> ```
> 
> I don't really like how abstracted it is but it is supposedly the preferred way to do spawns for generic buildings

Right I thought I'd worked on this before.   I'm wondering what the deal is there.

**Pull request closed without merging**